server/resources/resource_management_test.go

```diff
@@ -1428,6 +1429,57 @@ var _ = Describe("Resource manager", func() {
 			})
 
 			Describe("With extensions", func() {
+				Context("pre_update inserts zero-value", func() {
+					var (
+						requestMap map[string]interface{}
+					)
+					BeforeEach(func() {
+						requestMap = map[string]interface{}{
+							"test_string": "Steloj ne estas en ordo.",
+							"test_bool":   true,
+						}
+						events["pre_update"] = `if (context.resource.test_bool === undefined) { context.resource.test_bool = false; }`
```

Comment:
unnecessary

Reply:
why? It simulates Goext behavoir in JS code.